src/app/Plans/components/PlansTable.tsx

```diff
@@ -169,6 +184,13 @@ const PlansTable: React.FunctionComponent<IPlansTableProps> = ({
   return (
     <>
       <Level>
+        <LevelItem>
+          <FilterToolbar
+            filterCategories={filterCategories}
+            filterValues={filterValues}
+            setFilterValues={setFilterValues}
+          />
+        </LevelItem>
         <LevelItem>
           <CreatePlanButton variant="secondary" label="Create" />
```

Comment:
The placement of the Create button in the dead center of the page is a little weird, I think we should put it in that first LevelItem right next to the FilterToolbar. We probably want it vertically-aligned at the top instead of the middle (so it stays in the same place visually when the filter chips appear) and with a medium margin from the toolbar, but I'm not sure what will be needed exactly until we move it.

Reply:
@mturley @gildub The Create button should be 16 px to the right of the search field as documented here: https://www.patternfly.org/v4/components/toolbar/react

Reply:
@vconzola, I've re-positioned button. Please validate.